h/static/scripts/controllers/list-input-controller.js

```diff
@@ -23,7 +23,8 @@ class ListInputController extends Controller {
 
     // Handle 'Remove' button.
     element.addEventListener('click', (event) => {
-      if (event.target.getAttribute('data-ref') === 'removeItemButton') {
+      const btn = event.target.closest('button');
+      if (btn.getAttribute('data-ref') === 'removeItemButton') {
```

Comment:
This is needed because the user is now clicking in an `<svg>` (inside the button) rather than directly on the `<button>`.